apps/spruce/src/hooks/useTableSort/index.ts

```diff
@@ -21,35 +19,24 @@ type CallbackType = (sorter: SortingState) => void;
 export const useTableSort = (props?: Props): CallbackType => {
   const [queryParams, setQueryParams] = useQueryParams();
 
-  const tableChangeHandler = ((sorter: SortingState) => {
+  const tableChangeHandler = (sorter: SortingState) => {
     props?.sendAnalyticsEvents?.(sorter);
 
     const nextQueryParams = {
       ...queryParams,
       [PaginationQueryParams.Page]: "0",
       [TableQueryParams.Sorts]: undefined,
-      [TableQueryParams.SortDir]: undefined,
-      [TableQueryParams.SortBy]: undefined,
     };
 
-    if (sorter.length === 1) {
-      const { desc, id } = sorter[0];
-      nextQueryParams[TableQueryParams.SortDir] = desc
-        ? SortDirection.Desc
-        : SortDirection.Asc;
-      nextQueryParams[TableQueryParams.SortBy] = id;
-    } else if (sorter.length) {
-      const sortString = sorter
-        .map(({ desc, id }) =>
-          getSortString(id, desc ? SortDirection.Desc : SortDirection.Asc),
-        )
-        .filter(Boolean)
-        .join(";");
+    if (sorter.length) {
+      const sortArray = sorter.map(({ desc, id }) => ({
+        id,
+        direction: desc ? SortDirection.Desc : SortDirection.Asc,
+      }));
```

Comment:
(Just leaving this one comment since it affects a bunch of files) I'm not sure that we should make this change since it affects so many different tables. For example, sorting on the hosts table doesn't seem to work with these changes applied.

We could change the sorting later in [DEVPROD-4236](https://jira.mongodb.org/browse/DEVPROD-4236) if that's a preferable option